content/blog/redfish-interoperability-profiles.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Introduction to Redfish Interoperability Profiles
+date: 2024-07-03T10:58:04.215Z
+author: François Donzé
+authorimage: /img/fdz-photoprofile.png
+disable: false
+tags:
+  - Redfish
+  - ilo-restful-api
+---
+<style> li { font-size: 27px; line-height: 35px; max-width: none; } </style>
+
+<style> figcaption {font-style: italic; font-size: 15px; line-height: 33px; max-width: none;} </style>
+
+## Introduction
+
+When I explain to an audience that the <a href="https://redfish.dmtf.org" target="_blank">Redfish® standard</a> 
```

Comment:
looking at me with big eyes and this question in mind:  to be replaced by looking at me puzzled and asking themselves...